src/annotator/util/frame-util.js

```diff
@@ -46,18 +60,28 @@ export function isAccessible(iframe) {
   }
 }
 
+/**
+ * @param {HTMLIFrameElement} iframe
+ * @param {() => void} callback
+ */
 export function isDocumentReady(iframe, callback) {
```

Comment:
This is an existing issue, but it's not very clear from the function name and API what exactly `callback` does. A better API might be something like `waitForDocumentReady(iframe): Promise<void>`. Same with the function below.

Reply:
I can changed the name in a follow up PR.